3p/integration-lib.js

```diff
@@ -39,6 +39,7 @@ const AMP_EMBED_ALLOWED = {
   _ping_: true,
   '1wo': true,
   '24smi': true,
+  adskeeper: true,
```

Comment:
@powerivq do we still want this file?

Reply:
Yes, if the network wants amp-embed, this is still doable.